xrpl/testutil/integration/flag.go

```diff
@@ -0,0 +1,22 @@
+package integration
+
+import (
+	"os"
+	"testing"
+)
+
+const (
+	IntegrationEnvVar = "INTEGRATION"
+)
+
+// GetEnv returns the integration environment.
+// If the environment is not set, it skips the tests.
+// This function is intended to be used in tests that need to run against a specific environment.
+// Run it before creating the runner to retrieve the environment host and faucet provider.
+func GetEnv(t *testing.T) Env {
+	if _, ok := IntegrationEnvs[EnvKey(os.Getenv(IntegrationEnvVar))]; !ok {
```

Comment:
When do we need to use the `github.com/joho/godotenv` library to load env variables? I am asking because I tried recently and I needed this library to read from an `.env` file.

Reply:
There's no need to! the `os` native library let's you retrieve environment variables.